src/pages/ForumPage/ForumPage.tsx

```diff
@@ -74,17 +77,30 @@ const ForumPage: React.FC = () => {
         []
     )
 
+    const onChangeTheme = () => {
+        dispatch(
+            saveTheme({
+                id: user.id,
+                name: theme?.name === 'dark' ? 'light' : 'dark',
+            })
+        )
+    }
+
     return (
         <Layout hasMenu>
             <Container direction="column">
-                <ForumWrapperStyled>
+                <ForumWrapperStyled theme={theme?.name || 'dark'}>
                     <ForumTitleWrapStyled>
                         <ForumTitleStyled>Список форумов</ForumTitleStyled>
+                        <Switcher
+                            onChange={onChangeTheme}
+                            checked={theme?.name === 'dark' ? false : true}
```

Comment:
Можно сократить
```suggestion
                            checked={!(theme?.name === 'dark')}
```
или проще
```suggestion
                            checked={theme?.name === 'light'}
```